storage/StorageExample/Shared/ContentView.swift

```diff
@@ -19,6 +19,10 @@ import FirebaseAuth
 import FirebaseStorage
 import FirebaseStorageSwift
 
+#if os(macOS)
+  typealias UIImage = NSImage
+  typealias UIApplication = NSWorkspace
```

Comment:
I'd recommend not doing this since it's clever, but not extensible: If the app ever depends on an iOS or macOS specific image feature, the typealias will break. Instead, you could wrap this in an `Image` helper struct that references the correct platform-specific type below and then redefine methods on top of it.